scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java

```diff
@@ -971,7 +972,9 @@ public void updateAfterWorkflowTaskFinished(final InternalJob job, final Changed
             long jobId = jobId(job);
 
             JobInfo jobInfo = job.getJobInfo();
-
+            Map<String, Serializable> resultMap = job.getResultMap();
+            resultMap.putAll(result.getResultMap());
+            logger.info("resultMap: " + resultMap.toString());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "resultMap: " 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)